logzio-java-appenders.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's not add IntelliJ files into git and add them to .gitignore of this project